src/platform/forms-system/src/js/helpers.js

```diff
@@ -372,6 +372,7 @@ export function getNonArraySchema(schema, uiSchema = {}) {
       const schemaPropertyKeys = Object.keys(newSchema.properties);
       const newUiSchema = Object.assign({}, uiSchema);
       newUiSchema['ui:order'] = uiSchema['ui:order']?.filter(item => {
+        // check item === '*' here?
         return schemaPropertyKeys.includes(item);
```

Comment:
I thought about updating this helper function, but after looking at the unit test, it appears to be a bit more complicated since we're inside an array?